samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -278,6 +271,127 @@ Testing with nRF Connect for Desktop
 #. Swap roles.
    Set the application to the RX mode and the connected development kit to the TX mode.
 
+Direct Test Mode terminal connection
+------------------------------------
+
+To send commands to and receive responses from the development kit that runs the Direct Test Mode sample, connect to it with RealTerm in Windows or Minicom in Linux.
+
+The Bluetooth Low Energy DTM UART interface standard specifies the following configuration:
+
+* Eight data bits
+* No parity
+* One stop bit
+* No hardware flow control
+* A selection of bit rates from 9600 to 1000000, one of which must be supported by the DUT.
+  It might be possible to run other bit rates by experimenting with parameters.
+
+.. note::
+   The default bit rate of the DTM UART driver is 19200 bps, which is supported by most certified testers.
+
+You must send all commands as two-byte HEX numbers. 
+The responses must have the same format.
+
+Connect with RealTerm (Windows)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The RealTerm terminal program offers a graphical interface for setting up your connection.
+
+.. figure:: /images/realterm.png
```

Comment:
The first two pictures (echo.png and minicom.png) don't seem to be used anywhere - should remove.

Reply:
Ok removed